edx_proctoring/static/proctoring/js/views/proctored_exam_add_allowance_view.js

```diff
@@ -160,38 +205,37 @@ edx = edx || {};
                 });
             }
         },
-        selectExamAtIndex: function(index) {
-            var selectedExam = this.proctored_exams[index];
-
-            if (selectedExam.is_proctored) {
-                // Selected Exam is a Proctored or Practice-Proctored exam.
-                if (selectedExam.is_practice_exam) {
-                    $('#exam_type_label').text(gettext('Practice Exam'));
-                } else {
-                    $('#exam_type_label').text(gettext('Proctored Exam'));
-                }
-
-                // In case of Proctored Exams, we hide the Additional Time label and show the Allowance Types Select
-                $('#additional_time_label').hide();
-                $('select#allowance_type').val('additional_time_granted').show();
-            } else {
-                // Selected Exam is a Timed Exam.
-                $('#exam_type_label').text(gettext('Timed Exam'));
-
-                // In case of Timed Exams, we show the "Additional Time" label and hide the Allowance Types Select
-                $('#additional_time_label').show();
-                // Even though we have the Select element hidden, the backend will still be using
-                // the Select's value for the allowance type (key).
-                $('select#allowance_type').val('additional_time_granted').hide();
-            }
-            this.updateAllowanceLabels('additional_time_granted');
+        selectExamAtIndex: function(examID, examName) {
+            console.log(examID);
+            console.log(examName);
+            $('.exam_dropdown:visible').val("default");
+            $('.exam_dropdown:visible option[value=' + examID + ']').remove();
+            var createdTag = this.createTag(examName, examID);
+            console.log(createdTag);
+            $('#selected_exams').append(createdTag);
+            this.updateCss();
         },
         selectExam: function() {
-            this.selectExamAtIndex($('#proctored_exam')[0].selectedIndex);
+            this.selectExamAtIndex($('.exam_dropdown:visible').val(), $('.exam_dropdown:visible :selected').text());
         },
         selectAllowance: function() {
             this.updateAllowanceLabels($('#allowance_type').val());
         },
+        selectExamType: function () {
+            $('.close').each(function() {
+                $(this).trigger('click');
+            });
+            if($('#proctored_exam').is(":visible")){
+                $('#proctored_exam').hide();
+                $('#timed_exam').show();
+                $('#allowance_type option[value="review_policy_exception"]').remove();
+            } else {
+                $('#proctored_exam').show();
+                $('#timed_exam').hide();
+                $('#allowance_type').append(new Option(gettext('Review Policy Exception'), 'review_policy_exception'));              
```

Comment:
What kind of allowance_type is `Review Policy Exception`? Is that a multiplier or a time in minutes value?

Reply:
For Proctored exams instructors are able to input a review policy exception such as 'notes are allowed', so any text can be used for this input.